.ci/travis-functions.sh

```diff
@@ -18,4 +18,18 @@ function xdebug-enable() {
     if [[ -f "/tmp/xdebug.ini" ]]; then
         phpenv config-add /tmp/xdebug.ini
     fi
+}
+
+function get-infection-pr-flags() {
+    if [[ "${TRAVIS_PULL_REQUEST}" == "false" ]]; then
+        INFECTION_PR_FLAGS="";
+    else
+        git remote set-branches --add origin $TRAVIS_BRANCH;
+        git fetch;
```

Comment:
Do we need to fetch everything? We might aswell only fetch the branches we need here

Reply:
```bash
git clone --depth=50 --branch=remove https://github.com/infection/infection.git infection

git branch
# output 
* remove

git remote set-branches --add origin master
git fetch;

# From https://github.com/infection/infection
# * [new branch]      master     -> origin/master

git branch
# output
* remove
```

it doesn't fetch everything as I can see

Reply:
`git fetch` has no `--depth=` here

Reply:
Please never mind, `git fetch` should respect shallow clone

Reply:
My bad, i wrongly assumed `git fetch` would fetch all branches